infra/scoped/output.tf

```diff
@@ -1,15 +1,22 @@
-# API Gateway Keys
+# API Gateway endpoints
 
-output "api_key-dummy" {
-  value = aws_api_gateway_api_key.dummy.value
+output "api_identity_v1_endpoint" {
+  value = "https://${aws_api_gateway_domain_name.identity_v1.domain_name}/"
 }
 
-# Auth0 Clients
+# Auth0 hostname
 
-output "auth0_client-dummy-client_id" {
-  value = auth0_client.dummy_test.client_id
+output "auth0_endpoint" {
+  value = "https://${auth0_custom_domain.identity.domain}/"
 }
 
-output "auth0_client-dummy-client_secret" {
-  value = auth0_client.dummy_test.client_secret
+# Auth0 clients
+
+output "auth0_client_dummy-test" {
```

Comment:
The mixing of `_` and `-` in identifiers seem a bit odd. Any specific reason for it?

Reply:
Yeah, I try to use `_` in lieu of a space, and a hyphen as they would normally we used or in place of a `/` etc.

So `auth0_client_dummy-test` could be interpreted as `Auth0 Client dummy-test`.